{{cookiecutter.repo_name}}/docs/source/conf.py

```diff
@@ -82,73 +83,85 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path .
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'default'
+pygments_style = "default"
 
 
 # -- Options for HTML output -------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+html_theme = "mdanalysis_sphinx_theme"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+html_theme_options = {
+{% if cookiecutter.github_host_account == "MDAnalysis" %}
+    "mda_official": True,
+{% else %}
+    "mda_official": False,
+{% endif %}
+}
+
+{%- if cookiecutter.github_host_account != "MDAnalysis" %}
+# Set your logo and favicon here -- replace the placeholders!
```

Comment:
Ok maybe that invalidates my previous comment?

Working off my mobile phone so apologies if it's super easy to find - is this in the docs?